doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_srv.rst

```diff
@@ -164,41 +167,48 @@ Target illuminance level
 External event triggers
 -----------------------
 
-While the Timeout event is controlled internally, the On and Off events are produced by external behavior.
+While the Timeout event is controlled internally, the On, Occupancy On and Off events are produced by external behavior.
 
 On event
 ~~~~~~~~
 
-The On event lets the Light LC Server know that there is activity in the room.
-It can be generated by light switches and sensors.
+The On event can be generated by light switches.
 
 * Light switches can implement one of the following models to send On messages that turn the Light LC Server on:
 
   * The :ref:`bt_mesh_onoff_cli_readme` model -- which should publish to the Light LC Server's extended Generic OnOff Server model.
   * The :ref:`bt_mesh_light_ctrl_cli_readme` model -- which should publish Light OnOff Set messages to the Light LC Server.
 
-* Occupancy sensors can also trigger the On event, depending on the current state and occupancy mode:
+.. _bt_mesh_light_ctrl_srv_occupancy_on_event:
+
+Occupancy On event
+~~~~~~~~~~~~~~~~~~
+
+The Occupancy On event lets the Light LC Server know that there is activity in the room.
```

Comment:
Nitpick (feel free to discard this): 
Specify something like "activity in proximity to the node/sensor" instead of "in the room"? 
(What if the LLC Server is outside?)

Reply:
I like this suggestion, but then I need to fix other references to the room on this page, which I'm not sure how to fix:
https://github.com/nrfconnect/sdk-nrf/blob/d58aa2f15dd259bd9d1a37f7b478dae2fcb13b1a/doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_srv.rst?plain=1#L22
https://github.com/nrfconnect/sdk-nrf/blob/d58aa2f15dd259bd9d1a37f7b478dae2fcb13b1a/doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_srv.rst?plain=1#L88-L92
https://github.com/nrfconnect/sdk-nrf/blob/d58aa2f15dd259bd9d1a37f7b478dae2fcb13b1a/doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_srv.rst?plain=1#L308
https://github.com/nrfconnect/sdk-nrf/blob/d58aa2f15dd259bd9d1a37f7b478dae2fcb13b1a/doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_srv.rst?plain=1#L321

Reply:
Added